gimme

```diff
@@ -316,13 +364,16 @@ unset GOOS
 unset GOPATH
 unset GOROOT
 
+GIMME_VERSION_PREFIX="$(_realpath "${GIMME_VERSION_PREFIX}")"
+GIMME_ENV_PREFIX="$(_realpath "${GIMME_ENV_PREFIX}")"
```

Comment:
It's a minor nitpick, but I found this to be non-awesome:

``` bash
gimme 1.4 .
```
results in an env file with the following:

```
export GOROOT="./go1.4.darwin.amd64"
```